packages/sdk/src/adapters/multipeer/rules.ts

```diff
@@ -885,7 +886,7 @@ export const Rules: { [id in Payloads.PayloadType]: Rule } = {
     'set-behavior': {
         ...DefaultRule,
         synchronization: {
-            stage: 'create-actors',
+            stage: 'set-behaviors',
```

Comment:
So not sure what this change is for? Should the set-behavior rule stage be set to this?  Not sure what this has to do with sound.  I am not familiar with this code, so would like to be sure this is right since it seems to be touching behaviors here.